codex-rs/app-server-protocol/src/protocol/common.rs

```diff
@@ -380,44 +518,15 @@ pub enum ServerNotification {
     #[serde(rename = "account/rateLimits/updated")]
     #[ts(rename = "account/rateLimits/updated")]
     #[strum(serialize = "account/rateLimits/updated")]
-    AccountRateLimitsUpdated(RateLimitSnapshot),
+    AccountRateLimitsUpdated(v2::AccountRateLimitsUpdatedNotification),
 
     /// DEPRECATED NOTIFICATIONS below
-    /// Authentication status changed
     AuthStatusChange(v1::AuthStatusChangeNotification),
```

Comment:
Is this why we have to do the triple `#[]` thing on the other variants? Should this be a separate enum?